libimage/runtime.go

```diff
@@ -548,7 +549,11 @@ func (r *Runtime) ListImages(ctx context.Context, names []string, options *ListI
 		filters = append(filters, compiledFilters...)
 	}
 
-	return filterImages(images, filters)
+	registries, err := sysregistriesv2.UnqualifiedSearchRegistries(r.systemContextCopy())
+	if err != nil {
+		return nil, err
+	}
+	return filterImages(images, filters, append([]string{"localhost"}, registries...))
```

Comment:
Not sure but i think `filterImages` has a signature `func filterImages(images []*Image, filters []filterFunc, registries []string)` . I think this should fail with `too many arguments in call to filterImages` but haven't tried running with podman.

Reply:
no my bad this is correct i missed closing `)`